software-delivery-performance-terraform/test/fileutil.go

```diff
@@ -0,0 +1,47 @@
+package test
```

Comment:
What's this for?

Reply:
Terratest, the testing framework, I need to read the configuration files while running the tests to read the parameters. fileutil does that.

Reply:
Added a comment in the method